### PR TITLE
Feature/wmt 784/caseload overview data service

### PR DESCRIPTION
**Pull request by ddebella**
Ready for review

### CODE REVIEW

File: test/helpers/data/aggregated-data-helper.js [hunk 1]
```diff
@@ -192,9 +192,33 @@ var addOffenderManager = function (inserts) {
...
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 7, location: 'CUSTODY'}))
+
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 0, location: 'LICENSE'}))
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 1, location: 'LICENSE'}))
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 2, location: 'LICENSE'}))
       tiers.push(Object.assign({}, defaultTier, {tier_number: 3, location: 'LICENSE'}))
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 4, location: 'LICENSE'}))
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 5, location: 'LICENSE'}))
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 6, location: 'LICENSE'}))
+      tiers.push(Object.assign({}, defaultTier, {tier_number: 7, location: 'LICENSE'}))
```
Comment (tsheppard01) [inline test/helpers/data/aggregated-data-helper.js]: You can compress this into a forEach loop, look at how I've done it in the worker should be able to adapt that to here I think

File: app/services/data/get-team-caseload.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+const config = require('../../../knexfile').web
+const knex = require('knex')(config)
+module.exports = function (id, type) {
+  var table = "team_caseload_overview"
```
Comment (tsheppard01) [inline app/services/data/get-team-caseload.js]: Got a standard formatting error, need to use single quotes here

File: app/services/data/get-team-caseload.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            'grade_code as gradeCode',
+            'untiered',
+            'd2',
+            'd1',
+            'c2',
+            'c1',
+            'b2',
+            'b1',
+            'a',
+            'total_cases as totalCases')
```
Comment (tsheppard01) [inline app/services/data/get-team-caseload.js]: You need to bring back the link_is AS linkId also in this query

File: test/integration/services/data/test-get-team-caseload.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        expect(results[0].b1).to.eql(30)
+        expect(results[0].a).to.eql(30)
+        expect(results[0].totalCases).to.eql(5)
+      })
+  })
+
+  after(function () {
+    return dataHelper.removeInsertedData(inserts)
+  })
+})
```
Comment (dconey646) [inline test/integration/services/data/test-get-team-caseload.js]: 👍

File: test/integration/services/data/test-get-team-caseload.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+        expect(results[0].untiered).to.eql(30)
+        expect(results[0].d2).to.eql(30)
+        expect(results[0].d1).to.eql(30)
+        expect(results[0].c2).to.eql(30)
+        expect(results[0].c1).to.eql(30)
+        expect(results[0].b2).to.eql(30)
+        expect(results[0].b1).to.eql(30)
+        expect(results[0].a).to.eql(30)
+        expect(results[0].totalCases).to.eql(5)
+      })
```
Comment (tsheppard01) [inline test/integration/services/data/test-get-team-caseload.js]: 👍

File: test/helpers/data/aggregated-data-helper.js [hunk 6]
```diff
@@ -192,9 +192,33 @@ var addOffenderManager = function (inserts) {
...
+
```
Comment (dconey646) [inline test/helpers/data/aggregated-data-helper.js]: Same comment as Terrence's above

File: app/services/data/get-team-caseload.js [hunk 7]
```diff
@@ -0,0 +1,27 @@
+const config = require('../../../knexfile').web
+const knex = require('knex')(config)
+module.exports = function (id, type) {
```
Comment (tsheppard01) [inline app/services/data/get-team-caseload.js]: No reason to bring type in, it's not being used so think we can remove

File: test/helpers/data/aggregated-data-helper.js [hunk 8]
```diff
@@ -192,9 +192,12 @@ var addOffenderManager = function (inserts) {
...
       var tiers = []
-      tiers.push(Object.assign({}, defaultTier, {tier_number: 1, location: 'COMMUNITY'}))
-      tiers.push(Object.assign({}, defaultTier, {tier_number: 2, location: 'CUSTODY'}))
-      tiers.push(Object.assign({}, defaultTier, {tier_number: 3, location: 'LICENSE'}))
+      var locations = ['COMMUNITY', 'CUSTODY', 'LICENSE']
+      locations.forEach(function (location) {
+        for (var tierNumber = 0; tierNumber < 8; tierNumber++) {
+          tiers.push(Object.assign({}, defaultTier, {tier_number: tierNumber, location: location}))
+        }
+      })
```
Comment (tsheppard01) [inline test/helpers/data/aggregated-data-helper.js]: 👍

**Pull request merged**